sled-agent/src/services.rs

```diff
@@ -2852,17 +2853,68 @@ impl ServiceManager {
                         SwitchService::Tfport { pkt_source, asic } => {
                             info!(self.inner.log, "Setting up tfport service");
                             let mut tfport_config =
-                                PropertyGroupBuilder::new("config")
+                                PropertyGroupBuilder::new("config");
+
+                            tfport_config = tfport_config
+                                .add_property(
+                                    "host",
```

Comment:
These should really be `dpd_host` and `dpd_port` or `dendrite_host` and `dendrite_port`.  I know the command line options are just `host` and `port`, but that's not ideal either.